src/meta/src/hummock/manager/mod.rs

```diff
@@ -3018,7 +3018,10 @@ impl CompactionState {
             Some(compact_task::TaskType::Ttl)
         } else if guard.contains(&(group, compact_task::TaskType::Dynamic)) {
             Some(compact_task::TaskType::Dynamic)
+        } else if guard.contains(&(group, compact_task::TaskType::Tombstone)) {
```

Comment:
How about using a match statement here without the default arm? In this case, we will get a compile error if a task type is not handled here.

Reply:
rust does not provide a way to enumerate all posssible enum values so my comment won't work.